### ISSUE TITLE
What's your license?

### ISSUE THREAD

**Issue by ghost** [maxnowack/ytpodcast#1]

**Comment by maxnowack** [comment]
I don't have chosen a license at this time. What is the reason for your question?

**Comment by ghost** [comment]
When there is no chosen license, the default state (USA here) is an implicit copyright. Meaning no one is allowed to do anything at all with it without your permission including running it. The license is usually what tells people what permissions they're allowed to have and what conditions there are to follow regarding your code. Without one, the code you've put up here publicly is basically useless to... everyone.

This guy does a far better job than me at explaining this: 
http://blog.codinghorror.com/pick-a-license-any-license/

And here is a site that tries to make license choice easy: 
http://choosealicense.com/

**TL;DR** It's illegal to run or modify your code without each and every person asking you directly for permission because no license means no permission to touch your code

**Comment by maxnowack** [comment]
Thank you for the explanation, I've added GPLv2 :)

**Comment by ghost** [comment]
:)
